test/test.js

```diff
@@ -9,7 +9,7 @@ describe("loader", function() {
 			query: "?abc.def.ghi=>1"
 		}, "").should.be.eql(HEADER +
 			"var abc = (abc || {});\n" +
-			"abc.def = {};\n" +
+			"abc.def = abc.def || {};\n" +
```

Comment:
@michael-ciniawsky is correct. This should be two tests, the existing plus an additional test to cover the new nesting functionality